### ISSUE TITLE
CloudWorker and QueueWorker share tasks

### ISSUE THREAD

**Issue by malixsys** [soyuka/relieve#4]
```
// DB
    this.worker = relieve.workers.CloudWorker({});
    const task = new CallableTask(path.resolve(__dirname, 'database_task.js'), {restart: true});
    task.name = 'db';
    this.worker.add(task);
//RUNNER
    this.worker = relieve.workers.QueueWorker({ concurrency: 4 });
    if (this.worker.tasks.db) {
      // true!!??
    }
```

**Comment by soyuka** [comment]
Mhh this looks weird... 

Might it be that the line https://github.com/soyuka/relieve/blob/master/packages/relieve/workers/CloudWorker.js#L51 is messing up the scope?

**Comment by malixsys** [comment]
I think the
```
const tasks = new Map()
```
In Worker needs to be a per instance variable...

**Comment by soyuka** [comment]
thanks for the bug report @malixsys ! It should be fixed in 2.2.2!
